src/renderer/lens-app.tsx

```diff
@@ -36,6 +36,35 @@ import { registerIpcHandlers } from "./ipc";
 import { ipcRenderer } from "electron";
 import { IpcRendererNavigationEvents } from "./navigation/events";
 import { catalogEntityRegistry } from "./api/catalog-entity-registry";
+import { dsn, dsnIsValid, isProduction } from "../common/vars";
+import { CaptureConsole, Dedupe, Offline } from "@sentry/integrations";
+import * as Sentry from "@sentry/electron/dist/renderer";
+import { UserStore } from "../common/user-store";
+
+if (dsnIsValid) {
+  Sentry.init({
```

Comment:
try/catch?

Reply:
Added https://github.com/lensapp/lens/pull/3252/commits/1bcf1b17ea92191b3c48046ecce259fc6f98a89a to improve `dsnIsValid` check.